pixleesdk/src/main/java/com/pixlee/pixleesdk/PXLAlbum.java

```diff
@@ -214,4 +218,34 @@ public boolean openedWidget() {
         pxlClient.makeAnalyticsCall("events/openedWidget", body);
         return true;
     }
+
+    public boolean loadMore() {
+        PXLClient pxlClient = PXLClient.getInstance(context);
+        JSONObject body = new JSONObject();
+        StringBuilder stringBuilder = new StringBuilder();
+        int lastIdx = ((this.page - 1) * this.perPage);
+        for (int i = lastIdx; i < this.photos.size(); i++) {
+            try {
+                stringBuilder.append(this.photos.get(i).id);
+                if(i != this.photos.size() - 1){
+                    stringBuilder.append(",");
+                }
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+
+        try{
+            body.put("album_id",  Integer.parseInt(this.id));
+            body.put("per_page", this.perPage);
+            body.put("page", this.page);
+            body.put("photos", stringBuilder.toString());
```

Comment:
@firetix  if kb adds this to android it needs to get added to ios too

Reply:
I did https://github.com/pixlee/ios-sdk-carthage/pull/7/files#diff-3a24009da7c676cdd9b02eee9de5f1ddR289